src/wbi/timing.py

```diff
@@ -22,9 +24,17 @@ def _load_timing_file(self):
         # Simplify column names
         timing.columns = ["time", "frame"]
 
-        timing = timing.sort_values(by="frame").drop_duplicates(
```

Comment:
Dropping duplicates here instead of later would mean that we can no longer run the optimization of fixing the jump by 2 + jump by 0 => jump by 1 that we need to do later.